### PR TITLE
Add user-oriented resources in README.md

### PR DESCRIPTION
**Pull request by ffaf1**
Manual and venues where to (h)ask questions.

---
Please include the following checklist in your PR:

* [X] Patches conform to the [coding conventions](https://github.com/haskell/cabal/blob/master/CONTRIBUTING.md#other-conventions).
* [X] Any changes that could be relevant to users [have been recorded in the changelog](https://github.com/haskell/cabal/blob/master/CONTRIBUTING.md#changelog).
* [X] The documentation has been updated, if necessary.
* [X] ~~Include [manual QA notes](https://github.com/haskell/cabal/blob/master/CONTRIBUTING.md#qa-notes) if your PR relates to cabal-install~~ N/A.

~~Bonus points for added automated tests!~~ N/A

### CODE REVIEW

**Review by geekosaur — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -70,6 +70,16 @@ Ways to build `cabal-install` for everyday use
     if you don't have a pre-existing version of `cabal-install`,
     look into the [`bootstrap`](bootstrap) directory.
 
+Learn how to use `cabal` and support
```
Comment (ulysses4ever) [inline README.md:73]:
```suggestion
Learn how to use `cabal` and get support
```
@geekosaur would know better but I think the current phrasing is a bit clumsy in that it reads more like:

```
Learn [the following things]:

1. how to use `cabal`
2. support
```

**Review by ulysses4ever — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (geekosaur) [reply README.md:73]: Agreed

**Review by Kleidukos — APPROVED** [review]

**Comment by ulysses4ever** [comment]
@mergify rebase

**Comment by ulysses4ever** [comment]
@mergify rebase

**Pull request merged**